libs/video.js

```diff
@@ -10,9 +10,11 @@
 
 const ffmpeg = require(`fluent-ffmpeg`);
 const shortId = require(`shortid`);
+const fs = require('fs');
+var music = fs.readdirSync(`${__dirname}/../resources/sounds/`)
 
 const transition = `${__dirname}/../resources/videos/glitch.mp4`;
-const backgroundMusic = `${__dirname}/../resources/sounds/lofi1.mp3`;
+const backgroundMusic = `${__dirname}/../resources/sounds/` + music[Math.floor(Math.random() * music.length)];
```

Comment:
This will pick a random music file at import time, meaning if you generate multiple videos, they will all have that same music background. But this is good enough for now, I'll merge it.